sklearn/tree/tests/test_export.py

```diff
@@ -218,9 +219,19 @@ def test_graphviz_errors():
 
     clf.fit(X, y)
 
-    # Check feature_names error
-    out = StringIO()
-    assert_raises(IndexError, export_graphviz, clf, out, feature_names=[])
+    # Check feature_names less than number of features error
```

Comment:
--> Check if it errors when length of feature_names mismatches with number of features